src/Arcus.Security.Providers.HashiCorp/HashiCorpSecretProvider.cs

```diff
@@ -138,7 +138,7 @@ protected async Task<SecretData> GetTrackedSecretAsync(string secretName)
             };
 
             var isSuccessful = false;
-#if NET6_0
+#if NET6_0 || NET8_0
```

Comment:
Also here, I think we can remove the conditional compiler directive, since there's no situation in this version where we still refer to an arcus library that contains the `DependencyMeasurement` class ?

Reply:
Aha, think we can even simplify a lot more in the project files, too. Thx. 👍

Reply:
Removed the directives here: https://github.com/arcus-azure/arcus.security/pull/421/commits/d24b66901b7efe25e896f007b5b6a323722a58c4